--- xautomation-1.03/Makefile.am~	2009-04-14 18:06:23.000000000 +0200
+++ xautomation-1.03/Makefile.am	2012-02-08 22:20:30.579622481 +0100
@@ -1,7 +1,7 @@
 bin_PROGRAMS = xte rgb2pat png2pat visgrep pat2ppm patextract xmousepos
 man_MANS = pat2ppm.1 patextract.1 png2pat.1 rgb2pat.1 visgrep.1 xautomation.7 xmousepos.1 xte.1
 xte_SOURCES = xte.c debug.c kbd.h keysym_map.h multikey_map.h
-xte_LDADD = $(X_LIBS) $(X_EXTRALIBS) -lXtst
+xte_LDADD = $(X_LIBS) $(X_EXTRALIBS) -lXtst -lX11
 rgb2pat_SOURCES = rgb2pat.c image.c image.h debug.c debug.h
 rgb2pat_LDADD = -lpng
 png2pat_SOURCES = png2pat.c image.c image.h debug.c debug.h
